logstash/README.md

```diff
@@ -128,6 +128,7 @@ useful for the [http input plugin][], for instance.
 | `maxUnavailable`          | The [maxUnavailable][] value for the pod disruption budget. By default this will prevent Kubernetes from having more than 1 unhealthy pod in the node group                                                                          | `1`                                   |
 | `nameOverride`            | Overrides the chart name for resources. If not set the name will default to `.Chart.Name`                                                                                                                                            | `""`                                  |
 | `nodeAffinity`            | Value for the [node affinity settings][]                                                                                                                                                                                             | `{}`                                  |
+| `podAffinity`             | Value for the [pod affinity settings][]                                                                                                                                                                                              | `{}`                                  |
```

Comment:
This link isn't vaild, can you add a link like in https://github.com/ebuildy/helm-charts/blob/master/logstash/README.md#L219?

Reply:
done